x/collection/keeper/nft.go

```diff
@@ -4,15 +4,31 @@ import (
 	gogotypes "github.com/gogo/protobuf/types"
 
 	sdk "github.com/line/lbm-sdk/types"
-	sdkerrors "github.com/line/lbm-sdk/types/errors"
 	"github.com/line/lbm-sdk/x/collection"
 )
 
+func legacyNFTNotFoundError(k Keeper, ctx sdk.Context, contractID string, tokenID string) error {
+	// legacy
+	if err2 := collection.ValidateLegacyNFTID(tokenID); err2 == nil {
```

Comment:
`err2 == nil` is intentional, isn't it?
When I look at this first time, I thought it was a bug because we use `err != nil` a lot by convention.

Sorry for that I don't have any idea. Is there any way to express it in another way?

Reply:
We can change it into `err2 != nil` and use an `else` block, but it's against the common coding style.
As a last resort, I will leave a comment.